queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/README.md

```diff
@@ -48,13 +48,26 @@ The Default Queue project within the fiskaltrust Middleware system serves as a t
 
 - `Cases.cs`: Enumerates various cases for handling specific functionalities and scenarios within the system.
 
+## Naming Convention
+
+In this project, you may notice the use of "XX" in various class and file names. This "XX" serves as a placeholder representing a specific market or country code. When adapting the code for a particular market, replace "XX" with the appropriate market identifier.
```

Comment:
```suggestion
In this project, you may notice the use of "XX" and "DEFAULT" in various class and file names.

When using this as the basis for a new market _both_ "XX" and "DEFAULT" should be replaced with the market country code (e.g. "IT" for Italy).

The "DEFAULT" files include a working minimum implementation that is used in the DEFAULT Queue. This implementation needs to be adapted to the new market.

The "XX" files serve as a placeholders representing a feature that might be needed in a real market but not in the DEFAULT queue.
```

Reply:
Only a suggestion, but I would make that a bit clearer 😁

Reply:
Looks better, ye :) thanks for your suggestion. Already added